File: examples/java/kora-java-resilient/src/main/java/io/koraframework/example/resilient/TimeoutService.java
package io.koraframework.example.resilient;

import io.koraframework.common.annotation.Component;
import io.koraframework.resilient.timeout.annotation.Timeout;

@Component
public class TimeoutService {

    @Timeout(DefaultTimeouter.class)
    public String getSuccessful() {
        return "OK";
    }
}

File: examples/kotlin/kora-kotlin-resilient/src/main/kotlin/io/koraframework/kotlin/example/resilient/TimeoutService.kt
package io.koraframework.kotlin.example.resilient

import io.koraframework.common.annotation.Component
import io.koraframework.resilient.timeout.annotation.Timeout
import java.util.concurrent.ThreadLocalRandom

@Component
open class TimeoutService {
    @Timeout(DefaultTimeouter::class)
    open fun getSuccessful(): String = "OK"
}
